docs/_source/practical_guides/create_dataset.md

```diff
@@ -52,7 +52,61 @@ fields = [
 The order of the fields in the UI follows the order in which these are added to the `fields` attribute in the Python SDK.
 ```
 
-##### Define `questions`
+#### Define metadata properties
+
+Metadata properties allow you to configure the use of metadata information for the filtering and sorting features available in the UI and Python SDK.
+
+You can define metadata properties using the Python SDK by providing the following arguments:
+
+- `name`: The name of the metadata property, as it will be used internally.
+- `title` (optional): The name of the metadata property, as it will be displayed in the UI. Defaults to the `name` value, but capitalized.
+- `visible_for_annotators` (optional): A boolean to specify whether the metadata property will be accessible for users with an `annotator` role in the UI (`True`), or if it will only be visible for users with `owner` or `admin` roles (`False`). It is set to `True` by default.
```

Comment:
this is not really clear to me.

Reply:
You're right about questions, but perhaps before guidelines because it's good to make a reference of what the annotation team can find and use within the UI and for that you need to know what metadata will be available for them.

Reply:
Could you be more specific about what exactly doesn't feel clear to you? That way I can target the specific text and look for solutions :)